schemas/material_schema.json

```diff
@@ -145,7 +145,7 @@
                     "properties": {
                         "density": {
                             "type": "number",
-                            "description": "Density of the material in kg/m^3."
+                            "description": "Density of the material in kg/m^3^."
```

Comment:
Why the ^ at the end?

Reply:
Now it looks correct in Antora. 

Now
![image](https://github.com/user-attachments/assets/f9934d43-6ccf-4060-bf3c-3eba3a2f87f3)

Before: 
![image](https://github.com/user-attachments/assets/e35e7b53-d80e-48a5-9562-539461df6260)

Reply:
Ah nice!